docs-template/index.md

```diff
@@ -1,22 +1,22 @@
-# {Charm name} Operator
+# < Charm name > Operator
```

Comment:
This is a nitpick, I wouldn't put spaces so rather <Charm name> to reduce the amount of whitespace

Reply:
Sounds good, I'll make this change in the next PR.